script/FullDeployer.s.sol

```diff
@@ -54,12 +54,13 @@ contract FullDeployer is HubDeployer, SpokeDeployer {
         // Since `wire()` is not called, separately adding the safe here
         guardian.file("safe", address(adminSafe));
         saveDeploymentOutput();
-        vm.stopBroadcast();
 
         bool isNotMainnet = keccak256(abi.encodePacked(environment)) != keccak256(abi.encodePacked("mainnet"));
 
         if (!isNotMainnet) {
             removeFullDeployerAccess(msg.sender);
         }
+
+        vm.stopBroadcast();
     }
 }
```

Comment:
I think this change is needed to maintain the broadcast during the phase of removing the deployer access